template.tpl

```diff
@@ -767,104 +759,217 @@ ___TEMPLATE_PARAMETERS___
+                      },
+                      {
+                        "type": "TEXT",
+                        "name": "attributionExcludeReferrers",
+                        "displayName": "Exclude Referrers",
+                        "simpleValueType": true,
+                        "help": "Pass a comma-separated list of referring domains you want to exclude from campaign attribution. \u003ca href\u003d\"\"\u003eRead more\u003c/a\u003e."
+                      },
+                      {
+                        "type": "CHECKBOX",
+                        "name": "attributionResetSession",
+                        "checkboxText": "Reset the session on a new campaign",
+                        "simpleValueType": true
+                      }
+                    ],
+                    "enablingConditions": [
+                      {
+                        "paramName": "detAttribution",
+                        "paramValue": true,
+                        "type": "EQUALS"
+                      }
+                    ]
+                  }
+                ],
+                "enablingConditions": [],
+                "help": "Check this box to enable attribution tracking. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-marketing-attribution\"\u003eRead more\u003c/a\u003e."
+              },
+              {
+                "type": "CHECKBOX",
+                "name": "detPageViews",
+                "checkboxText": "Enable page views tracking",
+                "simpleValueType": true,
+                "subParams": [
+                  {
+                    "type": "TEXT",
+                    "name": "pageViewType",
+                    "displayName": "Page View Type",
+                    "simpleValueType": true,
+                    "defaultValue": "[Amplitude]  Page Viewed",
+                    "help": "The event type for page view event. The default value has been change from \u003cstrong\u003ePage View\u003c/strong\u003e to \u003cstrong\u003e[Amplitude]  Page Viewed\u003c/strong\u003e. If you want to keep it consistent, please use \u003cstrong\u003ePage View\u003c/strong\u003e as the default value."
+                  },
+                  {
+                    "type": "GROUP",
+                    "name": "pageViewOptions",
+                    "displayName": "",
+                    "groupStyle": "NO_ZIPPY",
+                    "subParams": [
+                      {
+                        "type": "SELECT",
+                        "name": "pageViewTrackOn",
+                        "displayName": "Page View trigger",
+                        "macrosInSelect": true,
+                        "selectItems": [
+                          {
+                            "value": "pageTrackOnPageLoads",
+                            "displayValue": "Page Loads (default)"
```

Comment:
Suggesting to use native GTM language

```suggestion
                            "displayValue": "Page View (default)"
```

Reply:
I would like to stick to what it is to avoid more customer confusion.

Reply:
I'm leaning more towards using the same terms across our products, especially we used "page view" in Marketing Analytics previously